Issue260 rom

mlauster opened the pull request:
This introduces the all-new reduced order models, thermal zone models and multizone models.
This closes #260 
This closes #78 
This closes #151

marcusfuchs commented on AixLib/DataBase/ThermalZones/OfficePassiveHouse/package.mo:
```diff
@@ -0,0 +1,5 @@
+within AixLib.DataBase.ThermalZones;
+package OfficePassiveHouse "Contains reocrds for an office as passive house"
```
@mlauster 
"reocrds" should be "records"

mlauster commented:
@marcusfuchs, I corrected the typo and added a User's Guide to `Buildings` package. The plan is to integrate `Buildings` into `ThermalZones` on the mid term, see #276. To have it more strict, I deleted the remaining example and the old VDI 6007 validation in `LOM`.
Regarding the order of the VDI 6007 validation cases, they are developed within the Annex60-library and the merging script leads to this kind of order. Let's see if we change it in the Annex60 library.

mlauster commented:
@marcusfuchs, all conflicts are solved, branch is ready to merge

PMehrfeld commented:
There is still a branch [issue260_ROM_eqAirTemp](https://github.com/RWTH-EBC/AixLib/tree/issue260_ROM_eqAirTemp).
#260 is already closed.
This PR only merged the branch `issue260_ROM`.
In #260 there is a reference to the still opened issue #276. 

If the branch [issue260_ROM_eqAirTemp](https://github.com/RWTH-EBC/AixLib/tree/issue260_ROM_eqAirTemp) obsolete, please delete it @mlauster.
